logos/src/logos/main.py

```
@@ -0,0 +1,349 @@
+    with DBManager() as db:
+        return db.get_api_id(data.logos_key, data.api_key)
+
+
+@app.post("/logosdb/get_role")
+async def get_role(data: GetRole):
+    with DBManager() as db:
+        return db.get_role(**data.dict())
+
+
+@app.post("/logosdb/get_providers")
+async def get_providers(data: LogosKeyModel):
```

Ruff (0.11.9)

`LogosKeyModel` may be undefined, or defined from star imports

(F405)